File: app/src/main/java/com/alirezaashrafi/pico/MainActivity.java
package com.alirezaashrafi.pico;

import android.graphics.Bitmap;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ImageView;
import android.widget.LinearLayout;
import android.widget.RelativeLayout;

import com.alirezaashrafi.library.Pico;
import com.alirezaashrafi.library.interfaces.OnBitmapLoad;

import java.io.File;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        initViews();
/*"https://blog.verseo.pl/wp-content/uploads/2017/05/flower-2-720x450.jpg"*/

        Pico pico = new Pico(this);
        pico.from(new File("/sdcard/img.jpg"))
                .into(new OnBitmapLoad() {
                    @Override
                    public void bitmap(Bitmap bitmap) {
                        Pico.with(MainActivity.this).from(bitmap)//.smartScale()
                                .smartScale()
                                .smartCache()
                                //.setBlur(10)
                                //.setColorFilter(getResources().getColor(R.color.colorAccent))
                                //.setBlur(10)

                                //.into(img7)
                                .into(img1)
                                .into(img2)
                                .into(img3)
                                .into(img4)
                                .into(img5)
                                //.into(root)
                                //.into(relativeLayout)
                                .into(img6).setScaleType(ImageView.ScaleType.CENTER_INSIDE);
                    }
                });

        //Pico.with(this).from("http://timc.a-gh.org/ui/code/background.jpg").smartScale().into(root);
    }

    private ImageView img7;
    private LinearLayout root;
    private ImageView img1;
    private ImageView img2;
    private ImageView img3;
    private ImageView img4;
    private ImageView img5;
    private ImageView img6;
    private RelativeLayout relativeLayout;

    public void initViews() {
        img7 = (ImageView) findViewById(R.id.img7);
        root = (LinearLayout) findViewById(R.id.root);
        img1 = (ImageView) findViewById(R.id.img1);
        img2 = (ImageView) findViewById(R.id.img2);
        img3 = (ImageView) findViewById(R.id.img3);
        img4 = (ImageView) findViewById(R.id.img4);
        img5 = (ImageView) findViewById(R.id.img5);
        img6 = (ImageView) findViewById(R.id.img6);
        relativeLayout = (RelativeLayout) findViewById(R.id.relativeLayout);
    }


}

File: app/src/main/java/com/alirezaashrafi/pico/MainActivity.kt
package com.alirezaashrafi.pico

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.alirezaashrafi.library.Pico
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {


    var imgLink1 = "https://raw.githubusercontent.com/alirezaashrafi/pico/master/images/img1.png"
    var imgLink2 = "https://raw.githubusercontent.com/alirezaashrafi/pico/master/images/img2.png"
    var imgLink3 = "https://raw.githubusercontent.com/alirezaashrafi/pico/master/images/img3.png"
    var imgLink4 = "https://raw.githubusercontent.com/alirezaashrafi/pico/master/images/img4.png"
    var imgLink5 = "https://raw.githubusercontent.com/alirezaashrafi/pico/master/images/img5.png"
    var imgLink6 = "https://raw.githubusercontent.com/alirezaashrafi/pico/master/images/img6.png"
    var logo = "https://raw.githubusercontent.com/alirezaashrafi/pico/master/app/src/main/res/drawable/logo.png"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Pico.with(this).from(imgLink1).setColorFilter("#333355").into(img1)
        Pico.with(this).from(imgLink2).setColorFilter("#333355").into(img2)
        Pico.with(this).from(imgLink3).setColorFilter("#333355").into(img3)
        Pico.with(this).from(imgLink4).setColorFilter("#333355").into(img4)
        Pico.with(this).from(imgLink5).setColorFilter("#333355").into(img5)
        Pico.with(this).from(imgLink6).setColorFilter("#333355").into(img6)


    }

}
